scripts/download-mapbox-gl-native-ios.sh

```diff
@@ -8,20 +8,9 @@ cd ios/
 if ! which curl > /dev/null; then echo "curl command not found. Please install curl"; exit 1; fi;
 if ! which unzip > /dev/null; then echo "unzip command not found. Please install unzip"; exit 1; fi;
 
-if [ ! -d temp ]; then
-    mkdir temp
-fi
-
-if [ -d ./RCTMapboxGL/Mapbox.bundle ]; then
-    rm -r RCTMapboxGL/Mapbox.bundle
-fi
-
-mkdir RCTMapboxGL/Mapbox.bundle
 
 curl -sS https://mapbox.s3.amazonaws.com/mapbox-gl-native/ios/builds/mapbox-ios-sdk-$VERSION.zip > temp.zip
```

Comment:
Since you’re only using the dynamic framework, make sure it’s installing the `-dynamic.zip` package so the download size is much smaller.